INSTALL.rst

```diff
@@ -95,7 +95,16 @@ code).
 which modifies how the wasm and glue code are built and how they behave. An ``-s``
 should precede each flag.
 
-For example, to generate modularized glue code, use:
```

Comment:
I would introduce the first example with:
`For example, to generate a HTML page, use:`